dev/src/lobster/tools.h

```diff
@@ -1037,6 +1037,22 @@ template<typename T> T ReadMemInc(const uint8_t *&p) {
     return dest;
 }
 
+template<typename T> void ReadVec(const uint8_t *&p, T &v) {
```

Comment:
Might as well call this `ReadVec64` since 64-bit serialized lengths are not exactly common in the wild.
In fact, if this function is not used lets leave it out all-together. Or maybe better, just make the index type a template argument.

Reply:
it is used by octree_bindings and new usages in the skeletal code(though that in term was discussed to go into flatbuffers)